.packit.yaml

```diff
@@ -134,3 +136,4 @@ jobs:
       - fedora-39
       - fedora-38
```

Comment:
worth to be done in its own patch, but you can drop fedora-38, it has gone EOL long time ago.

Reply:
Agreed. It's in my TODO to do so. I was even thinking in remove fedora-39 and added 41 and 42. WDYT @sandrobonazzola  too early?

Reply:
Let's wait another couple of months for that.